src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,140 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type State = {
+  title: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+  description: string;
+};
+
+export const NewMovie: React.FC<{ onAdd: (movie: Movie) => void }> = ({
+  onAdd,
+}) => {
+  const [count, setCount] = useState(0);
+  const [inputValue, setInputValue] = useState<State>({
```

Comment:
```suggestion
  const [inputValue, setInputValue] = useState<Movie>({
```
looks like we can simplify it